archive/0000-staking-contract.md

```diff
@@ -0,0 +1,176 @@
+- Proposal Name: staking-contract
+- Start Date: 2020-01-12
+- NEP PR: [nearprotocol/neps#0000](https://github.com/nearprotocol/neps/pull/0000)
+- Issue(s): link to relevant issues in relevant repos (not required).
+
+# Summary
+[summary]: #summary
+
+Provides standard and reference internal design for staking contract with delegation.
+
+# Motivation
+[motivation]: #motivation
+
```

Comment:
let's start with explanation why we need this standard. What does it bring to the community and why it is used.